demo/src/components/SrlComponent.js

```diff
@@ -342,7 +268,7 @@ class HierplaneVisualization extends React.Component {
   selectPrevVerb() {
     const nextIdx =
         this.state.selectedIdx === 0 ? this.props.trees.length - 1 : this.state.selectedIdx - 1;
-    this.setState({ selectedIdx: nextIdx });
+    this.setState({ defaultValue: nextIdx });
```

Comment:
Is this intended? I think it breaks the ability to select the previous verb :).

Reply:
Definitely not ;-)